python/src/trezorlib/tools.py

```diff
@@ -240,19 +299,19 @@ def wrapped_f(client, *args, **kwargs):
 ALL_CAP_RE = re.compile("([a-z0-9])([A-Z])")
 
 
-def from_camelcase(s):
+def from_camelcase(s: str) -> str:
     s = FIRST_CAP_RE.sub(r"\1_\2", s)
     return ALL_CAP_RE.sub(r"\1_\2", s).lower()
 
 
-def dict_from_camelcase(d, renames=None):
+def dict_from_camelcase(d: Union[dict, Any], renames: Optional[dict] = None) -> dict:
```

Comment:
`Union[dict, Any]` is actually `Any`

Reply:
Correct, that was a bad attempt to say "we expect mostly dict, but also anything else"

Reply:
Changed in 4cfe54a